docs/cep/sink/sink-types/websocket-server.md

```diff
@@ -0,0 +1,35 @@
+---
+title: websocket-server
+---
+
+The [WebSocket sink](websocket.md) sends events through the server which are then passed to the `example` stream. All events received by `example` stream are then sent to the WebSocket server
+
+Clients can connect to `ws://localhost:9025/abc` to receive events from the stream.
```

Comment:
@StoyanTc Does the URL have to be this specific value (`ws://localhost:9025/abc`)?  Or do we need to provide more specific instructions for connecting?